docs/io-handbook/technology-radar.md

```diff
@@ -0,0 +1,134 @@
+
+#### Hold
+
+- [Log aggregation for business analytics](https://www.thoughtworks.com/radar/techniques?blipid=202005090)
+- [Microservice envy](https://www.thoughtworks.com/radar/techniques/microservice-envy)
+- [Templating in YAML](https://www.thoughtworks.com/radar/techniques/templating-in-yaml)
+
+## Tools
+
+#### Adopt
+
+- [Airflow](https://www.thoughtworks.com/radar/tools/airflow)
+- [Dependabot](https://www.thoughtworks.com/radar/tools/dependabot)
+- [ESLint](https://www.thoughtworks.com/radar/tools/eslint)
+- [Parcel](https://www.thoughtworks.com/radar/tools/parcel)
+- [Prettier](https://www.thoughtworks.com/radar/tools/prettier)
+- [Snyk](https://www.thoughtworks.com/radar/tools/snyk)
+- [Terraform](https://www.thoughtworks.com/radar/tools/terraform)
+- [Terragrunt](https://www.thoughtworks.com/radar/tools/terragrunt)
+- [Visual Studio Code](https://www.thoughtworks.com/radar/tools/visual-studio-code)
+- [Yarn](https://www.thoughtworks.com/radar/tools/yarn)
+
+#### Trial
+
+- [Headless Chrome for front-end test](https://www.thoughtworks.com/radar/tools/headless-chrome-for-front-end-test)
+- [tfsec](https://www.thoughtworks.com/radar/tools/tfsec)
+- [Terratest](https://www.thoughtworks.com/radar/tools/terratest)
+- [Visual Studio Live Share](https://www.thoughtworks.com/radar/tools/visual-studio-live-share)
+
+#### Assess
+
+- [Mountebank](https://www.thoughtworks.com/radar/tools/mountebank)
+- [WireMock](https://www.thoughtworks.com/radar/tools/wiremock)
+
+#### Hold
+
+- [CircleCI](https://www.thoughtworks.com/radar/tools/circleci)
+
+## Platforms
+
+#### Adopt
+
+- [Azure](https://www.thoughtworks.com/radar/platforms/azure)
+- [Azure DevOps](https://www.thoughtworks.com/radar/platforms/azure-devops) and [Azure Pipelines](https://www.thoughtworks.com/radar/platforms/azure-pipelines)
+- [CosmosDB](https://www.thoughtworks.com/radar/platforms/cosmos-db)
+- Azure Durable Functions
+- [Node](https://www.thoughtworks.com/radar/platforms/node-overload)
+
+#### Trial
+
+- [Netlify](https://www.thoughtworks.com/radar/platforms/netlify)
+
+#### Assess
+
+#### Hold
+
+- [Low-code platforms](https://www.thoughtworks.com/radar/platforms/low-code-platforms)
+- [Overambitious API gateways](https://www.thoughtworks.com/radar/platforms/overambitious-api-gateways)
+
+## Languages & Frameworks
+
+#### Adopt
+
+- [TypeScript](https://www.thoughtworks.com/radar/languages-and-frameworks/typescript)
+- [io-ts and fp-ts](https://www.thoughtworks.com/radar/languages-and-frameworks/io-ts)
+- [React Hooks](https://www.thoughtworks.com/radar/languages-and-frameworks/react-hooks)
+- Bootstrap
+
+#### Trial
+
+- [Redux and Redux Saga](https://www.thoughtworks.com/radar/languages-and-frameworks/redux)
+- [React Testing Library](https://www.thoughtworks.com/radar/languages-and-frameworks/react-testing-library)
+
+#### Assess
+
+- [Recoil](https://www.thoughtworks.com/radar/languages-and-frameworks/recoil)
```

Comment:
```suggestion
- [Recoil](https://www.thoughtworks.com/radar/languages-and-frameworks/recoil)
- [Jotai](https://github.com/pmndrs/jotai)
- [Zustand](https://github.com/pmndrs/zustand)
```


recoil is not suited for production use, zustand is

Reply:
Also the react-native integration is not complete.